noxfile.py

```diff
@@ -35,90 +36,14 @@
 reset_coverage = True
 
 
-REQUIREMENTS = {
-    ".": "requirements.txt",
-}
-for path in Path("requirements").iterdir():
-    if match := re.fullmatch("requirements_(.+).txt", path.name):
-        REQUIREMENTS[match.group(1)] = str(path)
-
-
-def depends(
-    *deps: str,
-    install_cwd: bool = False,
-    update: bool = True,
-) -> Callable[[NoxSessionFunc[P, T]], NoxSessionFunc[P, T]]:
-    """A session decorator that invokes :func:`.install` with the given parameters before running the session."""
-
-    def decorator(f: NoxSessionFunc[P, T]) -> NoxSessionFunc[P, T]:
-        @functools.wraps(f)
-        def wrapper(session: nox.Session, *args: P.args, **kwargs: P.kwargs) -> T:
-            install(session, *deps, update=update, install_cwd=install_cwd)
-            return f(session, *args, **kwargs)
-
-        return wrapper
-
-    return decorator
-
-
-def install(
-    session: nox.Session,
-    *deps: str,
-    run: bool = False,
-    install_cwd: bool = False,
-    update: bool = True,
-) -> None:
-    """
-    Installs dependencies in a session.
-    Dependencies from the main ``requirements.txt`` will always be installed.
-
-    Parameters
-    ----------
-    *deps: :class:`str`
-        Dependency group names, e.g. ``dev`` for ``requirements_dev.txt``.
-    run: :class:`bool`
-        Whether to use :func:`nox.Session.run` instead of :func:`nox.Session.install`,
-        useful to avoid warnings when running in the global python environment.
-    install_cwd: :class:`bool`
-        Whether the main package should be installed (in editable mode, i.e. ``-e .``).
-    update: :class:`bool`
-        Whether packages should be updated (i.e. ``-U``). Defaults to ``True``.
-    """
-
-    install_args = []
-
-    if update:
-        install_args.append("-U")
-    if install_cwd:
-        install_args.extend(["-e", "."])
-
-    for d in dict.fromkeys((".", *deps)):  # deduplicate
-        install_args.extend(["-r", REQUIREMENTS[d]])
-
-    if run:
-        session.run("python", "-m", "pip", "install", *install_args)
-    else:
-        session.install(*install_args)
-
-
-def is_venv() -> bool:
-    # https://stackoverflow.com/a/42580137/5080607
-    return (
-        # virtualenv < v20
-        hasattr(sys, "real_prefix")
-        # virtualenv >= v20, others
-        or sys.base_prefix != sys.prefix
-    )
-
-
-@nox.session()
-@depends("docs")
+@nox.session(python="3.8")
```

Comment:
not every system has a 3.8 interpreter, we shouldn't force a specific version for these tasks